wled00/FX.cpp

```diff
@@ -2639,9 +2658,11 @@ uint16_t WS2812FX::mode_bouncing_balls(void) {
   
   bool hasCol2 = SEGCOLOR(2);
   fill(hasCol2 ? BLACK : SEGCOLOR(1));
+  //fill(hasCol2 ? BLACK : SEGCOLOR(1));
+  // for (int i = 0; i < SEGLEN; i++) {if (random(10) > 5)blendPixelColor(i, BLACK, 100); }  // perhaps add a comet tail option in the future?
   
   for (uint8_t i = 0; i < numBalls; i++) {
-    float timeSinceLastBounce = (time - balls[i].lastBounceTime)/((255-SEGMENT.speed)*8/256 +1);
+    float timeSinceLastBounce = (time - balls[i].lastBounceTime)/((255-SEGMENT.speed)*15/256 +1);
```

Comment:
I think it was just personal preference. If I remember (months ago) I found the speed of the old effect too fast.
Not really an important change. 
Also, maybe because my original post suggested slowing the effect all the way down to demo the Anti Aliasing.
Who knows